server/fleet/microsoft_mdm.go

```diff
@@ -921,6 +923,30 @@ type SyncMLCmd struct {
 	SystemOrigin bool      `xml:"-"`
 }
 
+// ParseWindowsMDMCommand parses the raw XML as a single Windows MDM command.
+// A single <Exec> command is accepted as input.
+func ParseWindowsMDMCommand(rawXMLCmd []byte) (*SyncMLCmd, error) {
```

Comment:
~~Apologies if I'm missing something obvious, but do we have unit tests for this already?~~ 

I see this is covered in part by `cmd/fleetctl/mdm_test` so the unit tests would be somewhat superfluous. It might be nice to have some anyway if only for providing clear examples of what valid and invalid payloads look like in close proximity to the type (which is what prompted my question as I was reviewing). But I'm fine either way.

Reply:
Yeah good call, I'll add some!